packages/ds-healthcare-gov/src/styles/components/_FormLabel.scss

```diff
@@ -14,3 +14,9 @@ $error-message-icon-size: 28px;
 .ds-c-field__error-message--inverse::before {
   background-image: url('#{$image-path}/error-light-red.svg');
 }
+
```

Comment:
This was added to give backward compatibility to anyone who might be using this class on healthcare.gov without using the component. The override will only be applied to the component version in order to not display the icon twice.

Reply:
I think this works for backwards compatibility. Hopefully we can remove it in the future 🤞🏻

Reply:
Maybe we should explain this in the comments so we know when it can be removed.